src/model/file.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) 2022 spdx contributors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from enum import Enum, auto
+from typing import Optional, List, Union
+
+from src.model.checksum import Checksum
+from src.model.license import License
+from src.model.license_expression import LicenseExpression
+from src.model.spdx_no_assert import SpdxNoAssert
+from src.model.spdx_none import SpdxNone
+
+
+class FileType(Enum):
+    SOURCE = auto()
+    BINARY = auto()
+    ARCHIVE = auto()
+    APPLICATION = auto()
+    AUDIO = auto()
+    IMAGE = auto()
+    TEXT = auto()
+    VIDEO = auto()
+    DOCUMENTATION = auto()
+    SPDX = auto()
+    OTHER = auto()
+
+
+class File:
+    # Mandatory properties
+
+    spdx_id: str
+    name: str
+    checksums: List[Checksum]
+
+    # Optional properties
+
+    comment: Optional[str]
```

Comment:
I would keep the order of the properties from the spec; or is there some reason to have `comment` as the first field?

Reply:
I reordered the properties. It doesn't really work for the constructor though, because arguments without a default must come before arguments with a default

Reply:
yeah, well, that's just how it is